bin/start_fe.sh

```diff
@@ -176,6 +176,12 @@ if [ ! -d $LOG_DIR ]; then
     mkdir -p $LOG_DIR
 fi
 
+meta_dir="$STARROCKS_HOME/meta"
+read_var_from_conf meta_dir $STARROCKS_HOME/conf/fe.conf
+if [ ! -d $meta_dir ]; then
+    mkdir -p $meta_dir
+fi
```

Comment:
```suggestion
mkdir -p ${meta_dir:-"$STARROCKS_HOME/meta"}
```

Reply:
`export_env_from_conf` only parses uppercase variables, so `$meta_dir` is empty and `mkdir -p ${meta_dir:-"$STARROCKS_HOME/meta"}` is always `mkdir -p $STARROCKS_HOME/meta`

Reply:
ok

Reply:
so you can do
```
read_var_from_conf meta_dir $STARROCKS_HOME/conf/fe.conf
mkdir -p ${meta_dir:-"$STARROCKS_HOME/meta"}
```